apps/hackathon/src/app/global.css

```
@@ -0,0 +1,4 @@
+@import "tailwindcss";
+
+@source '../../../../libs/component-library';
+@import "../../../../libs/component-library/src/styles/globals.css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)